ui/src/views/Swagger.vue

```diff
@@ -0,0 +1,56 @@
+<template>
+  <div id="swagger-ui"></div>
+</template>
+
+<script>
+import axios from "axios";
+import SwaggerUI from "swagger-ui";
+import "swagger-ui/dist/swagger-ui.css";
+
+export default {
+  name: "swagger-ui",
+  mounted: function () {
+    const swagger = SwaggerUI({
+      dom_id: "#swagger-ui",
+      url: "/swagger/swagger.json",
+    });
+
+    axios.get("/oauth2id").then((response) => {
+      swagger.initOAuth({
+        ...(response.status == 200 && {
+          clientId: response.data["oauth2-client-id"],
+        }),
+        appName: "Workflow Launcher",
+        scopes: "openid email profile",
+      });
+    });
```

Comment:
If the response from `/oauth2id` is 200 then we spread the returned client ID into the parameter object.

If the backend was down I can't imagine that authenticating would be much help, but this is at least controlled behavior that doesn't just crash the swagger page if only the frontend is online.